src/components/PopUpBar/PopUpBar.css

```diff
@@ -1,5 +1,5 @@
 .popup_container {
-  position: flex;
+  position: fixed;
```

Comment:
Hi @LeeSingChanYau

you can add

top: 0%;
left: 0%;
here to make it stick to the top

Reply:
@Theophileaseh thanks for the feedback! Please review again.

Reply:
Thank you. Please ignore my comment on the other PR and keep it as you have at the top. Sounds like that's everyone's preference, so I agree.

Reply:
https://user-images.githubusercontent.com/67836976/219539353-4d38c1ab-06d2-467c-83ec-0d53cec7a719.mov